README.md

```diff
@@ -19,67 +19,21 @@ Docs at [https://dimagi.github.io/commcare-cloud/](https://dimagi.github.io/comm
 
 
 # Install and setup
-You will need python 2.7 and `virtualenvwrapper` installed
-to follow these instructions.
-
-Clone the commcare-cloud repo
-(suggested location alongside commcare-hq, if you have that repo cloned as well):
-
-```
-git clone https://github.com/dimagi/commcare-cloud.git
-cd commcare-cloud
-```
-
-Now make a virtualenv for ansible:
-
-```
-mkvirtualenv ansible
-```
-
-If you want `workon ansible` to always bring you to this directory, then you can also run
-
-```
-setvirtualenvproject
-```
-
-at this time.
-
-**Note**: If you already have commcare-cloud cloned, then just enter that directory
-and update it with
+You will need python 2.7 and `virtualenvwrapper` installed to follow these instructions:
 
 ```
-git pull
+sudo apt-get install git python-dev python-pip
+sudo pip install virtualenv virtualenvwrapper
 ```
 
-# Install commcare-cloud
-
-You must be in your virtualenv for the install script to work
+# Quick setup
+This should be run from your home directory:
 ```
-workon ansible
-```
-
-Then, simply run:
-
-```
-./install.sh
+source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
```

Comment:
@snopoke @dannyroberts looks like this needs some testing.
```
dmiller@linsrv:~$ virtualenv cchq-venv
New python executable in /home/dmiller/cchq-venv/bin/python
Installing setuptools, pip, wheel...done.
dmiller@linsrv:~$ . cchq-venv/bin/activate
(cchq-venv) dmiller@linsrv:~$ source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
Could not open requirements file: [Errno 2] No such file or directory: '/dev/requirements.txt'
Downloading dependencies from galaxy and pip
The program 'ansible-galaxy' is currently not installed. To run 'ansible-galaxy' please ask your administrator to install the package 'ansible'
Requirement already up-to-date: pip in /home/dmiller/cchq-venv/lib/python2.7/site-packages
[1]-  Done                    pip install pip --upgrade
[2]+  Exit 127                ansible-galaxy install -r ${COMMCARE_CLOUD_REPO}/ansible/requirements.yml
-bash: ./control/check_install.sh: No such file or directory
-bash: /home/dmiller/.commcare-cloud/repo/control/.bash_completion: No such file or directory
Do you want to have the CommCare Cloud environment setup on login?

You can always set it up later by running this command:
'[ -t 1 ] && source ~/init-ansible' >> ~/.profile

Welcome to Ansible Control

Available commands:
update-code - update the ansible repositories (safely)
workon ansible - activate the ansible virtual env
ansible-deploy-control [environment] - deploy changes to users on this control machine
commcare-cloud - CLI wrapper for ansible.
                 See commcare-cloud -h for more details.
                 See commcare-cloud <env> <command> -h for command details.

Deprecated Commands
ap - Use commcare-cloud <env> ap instead.
aps - Use commcare-cloud <env> aps instead.
ae - Use commcare-cloud <env> run-shell-command instead.
(cchq-venv) dmiller@linsrv:/dev$
```
Note that it left me in `/dev` (odd directory to end up in). Did I do something wrong?

Reply:
@millerdev this was a result of https://github.com/dimagi/commcare-cloud/pull/1414 i think and not this PR. I have https://github.com/dimagi/commcare-cloud/pull/1421 open to fix.